lib/response-handlers/initial-response-handler.js

```diff
@@ -0,0 +1,16 @@
+const handleResponse = (response) => {
+  return response.json()
+    .then(json => {
+      if (!response.ok) {
+        const error = {
+          status: response.status,
+          statusText: response.statusText,
+          json,
```

Comment:
@kat3kasper was hoping to get your thoughts on this line of code. I tried using the spread operator on this line...
```...json```
but I was getting an error saying unexpected token. Does that mean my browser or, maybe, some library I have is not up to date?

Reply:
Hey, @adrian-lara, can you tell us a little bit more about what you've tried? Did you Google that particular error and find any additional information?

Thanks!

Reply:
@s-espinosa , here's the article I looked at and made me phrase the question the way I did.
 (Sorry, should've shared the first time)
https://github.com/JeffreyWay/laravel-mix/issues/76

So, I followed the steps in the second response of this issue, but that didn't seem to do the trick. Notably, I used the spread operator in class the other day and in one of the homework exercises earlier this week, which is why I thought this app might need updating. But maybe I'm updating the wrong thing or doing something else wrong?

Also, this issue isn't holding us back, but I thought it was weird that a method I used in a different app wasn't working in this app.

Reply:
@adrian-lara great question that is pretty weird. I assumed it would be in the `preset-env` config for babel but maybe not! I love that you are trying to use this for error handling. Let meet on Monday and see if we can get this working